src/yoroi-wallets/cardano/shelley/transactions.ts

```diff
@@ -387,6 +387,8 @@ export const newAdaUnsignedTxFromUtxo = async (
   {
     // recall: we might have some implicit input to start with from deposit refunds
     const implicitSum = await txBuilder.getImplicitInput()
+    const foo = (await implicitSum.multiasset()).hasValue()
+    console.log('QWE', foo)
```

Comment:
```suggestion
```

Reply:
whoops, that was from debugging with Pedro, lol